04LessonFour/in-class-project/css/style.css

```diff
@@ -1,84 +1,29 @@
-body {
-    background-color: skyblue;
-}
-
-.head {
-    width: 100%;
-    height: 200px;
-    border: solid 3px black;
-}
-
-.head img {
-    height: 100px;
-    width: 100px;
-    margin-left: 655px;
-    border: solid 2px black;
-}
-
-.head h1 {
-    text-align: center;
-    margin-top: 10px;
-}
-
-nav a {
-    font-family: sans-serif;
-    font-size: 15px;
-    color: white;
-    text-align: center;
-    margin-left: 300px;
-}
-
-#two {
-    border: solid 2px black;
-    width: 60%;
-    float: left;
-}
-
-#two h2 {
-    text-align: center;
-}
-
-#two h3 {
-    text-decoration: underline;
-}
-
-#two img {
-    width: 50%;
-    height: 50%;
-    padding-bottom: 5px;
-    padding-left: 5px;
-}
-
-#three {
-    width: 25%;
-    height: 150px;
-}
-
-#three h3 {
-    text-decoration: underline;
-}
-
-.right img {
-    height: 20%;
-    width: 20%;
-    margin-left: 90px;
-    margin-top: 100px;
-}
-
-.right h3 {
-    font-family: monospace;
-    font-size: 18px;
-    color: black;
-    text-align: center;
-    margin-right: 100px;
-}
-
-.right p {
-    text-align: center;
-    margin-right: 100px;
-}
-
-.supes {
-    width: 10%;
-    height: 10%;
-}
\ No newline at end of file
+body {
+  margin: auto;
+  width: 80%;
+  font-family: "arial narrow", Arial, sans-serif;
+}
+h1, h2, h3{
+  font-weight: normal;
+}
+#heading {
+  background-color: #0071c5;
+  padding: 10px;
+  color: white;
+}
+nav a{
+  color: white;
+}
+h1{
+  font-size: 50px;
+  margin-bottom: 5px;
+}
+.support {
+  background-color: lightgrey;
+  margin-top: -9px;
+  padding: 10px;
+  margin-right: 4px;
+}
+.support h2{
+  margin-top: 2px;
+}
```

Comment:
you need to add:

img {
width: 100%;
}

to get your image to take up the entire table data cell.